src/targets/symbolCollector.ts

```diff
@@ -91,18 +91,22 @@ export class SymbolCollector extends BaseTarget {
         })
       );
 
-      await spawnProcess(SYM_COLLECTOR_BIN_NAME, [
-        '--upload',
-        'directory',
-        '--path',
-        dir,
-        '--batch-type',
-        this.symbolCollectorConfig.batchType,
-        '--bundle-id',
-        bundleId,
-        '--server-endpoint',
-        this.symbolCollectorConfig.serverEndpoint,
-      ]);
+      this.logger.info(
+        (
+          await spawnProcess(SYM_COLLECTOR_BIN_NAME, [
```

Comment:
Can we make sure we extract into a variable - IMO makes it easier to read.

```
const stdOut = await spawnProcess(...)
if (stdOut) {
  // we don't wanna log out undefined
  this.logger.info(stdOut); 
} else {
  // idk how to handle this case
}
```

Also from what I can see from https://github.com/getsentry/craft/blob/master/src/utils/system.ts#L127, the promise always resolves to a buffer, never undefined. Why is it typed that way?

Reply:
Oh wait sorry didn't see the early return case.

Reply:
Then we can make it:

```ts
const stdOut = await spawnProcess(...)
if (stdOut) {
  this.logger.info(stdOut); 
}
```